OpenRA.Game/Platform.cs

```diff
@@ -63,11 +63,26 @@ public static string RuntimeVersion
 			}
 		}
 
-		public static string SupportDir { get { return supportDir.Value; } }
-		static Lazy<string> supportDir = Exts.Lazy(GetSupportDir);
+		public static string SupportDir { get; private set; }
 
-		static string GetSupportDir()
+		public static void Initialize(Arguments args)
 		{
+			SupportDir = FindSupportDir(args);
+		}
+
+		static string FindSupportDir(Arguments args)
+		{
+			// Check for support directory overrides
+			if (args.Contains("SupportDir"))
```

Comment:
This will need some way to be picked up by the utility so that it can be added to the man page.